### PR TITLE
Microverse JS project 2: Restaurant Page

### PR DESCRIPTION
**Pull request by RailonA**
This project is part of Javascript Module with the aim of building a simple Restaurant page using the main features of ES6 and Webpack

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+const homeLink = document.querySelector("#home-link");
+const menuLink = document.querySelector("#menu-link");
+const contactLink = document.querySelector('#contact-link');
+
+DisplayHome.displayHome();
+
+
+// DisplayHome.card("./images/logo.png","Logo","this is the description")
+// Display Menu Page  displayMenu();
+// MenuLink.addEventListener("click", DisplayMenu.displayMenu())
```
Comment (uche-inyama) [inline src/index.js]: Remove commented lines of code.

File: src/menu.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
+/* eslint-disable */ 
+export var DisplayMenu = (function () {
+
+
+                return {
+                    displayMenu: function () {
+
+      
+                  
+                            // const main = document.querySelector('main');
```
Comment (uche-inyama) [inline src/menu.js]: enable eslint and remove commented line of code

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */ 
+
```
Comment (uche-inyama) [inline webpack.config.js:2]: enable eslint and fix the error flagged by eslint.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi team :slightly_smiling_face: 

I must commend your efforts so far, however, there is still a need for improvement.

* [x]  In some of your files, you have disabled eslint, kindly enable eslint in those files and fix the error flagged by eslint.
* [x]  There are also some commented-out lines of codes in your files, kindly remove them.
* [x]  Also, your files need to be properly formatted, especially the indentation.
* [x]  The contact page needs to be revised, it is not rendering properly on the webpage; it has a horizontal scroll bar.


#### That's it, make these changes, then submit another form for a code review.

### Happy coding :man_technologist: 

#### ``` Uchechukwu Henry Inyama ```

**Comment by mimipeshy** [comment]
## STATUS : CHANGES REQUESTED 🔴 ♻️ 
Hello @RailonA and @carloso0114 👋🏾 👋🏾 congratulations on making it this far 😄 . 

#### Highlights
- You have worked on some of the feedback given by the previous TSE ✔️ 

#### To improve
The following are yet to be implemented ⬇️ 
- [x]  In some of your files like index.js, you have disabled eslint, kindly enable eslint in those files and fix the error flagged by eslint.
- [x] The contact page needs to be revised, it is not rendering properly on the webpage; it has a horizontal scroll bar. Below is the image of how it renders ⬇️ 
image
- [x] The previous TSE asked you to enable eslint in webpack.config but didn't do so, kindly work on all changes requested

That's all 😄, kindly submit another review when done
If you need any clarification, kindly post your comment here and tag me @mimipeshy and will be glad to help

Regards,
Peris Ndanu

File: webpack.config.js (hunk 3 above)
Comment (mimipeshy) [reply webpack.config.js:2]: - [ ] You didn't enable eslint as suggested by the previous TSE

**Review by mke2111 — APPROVED** [review]
# 😄 🥇  STATUS: APPROVED  🥇 👏🏽 

## Hey Happy Coder, 👋🏽 
Congratulations, 🥂 . You got minimum requirements for approval. 👍🏽 

### **_Well Done_** 💯 

    1. The contact page is now well-positioned
    2. The linter checks run just fine. 



### You may now merge to the develop branch and remember to submit a project completion form.
### I wish you the very best in your upcoming projects.

**Cheers 🍹 and Happy coding 💻** 

<hr>

### Best regards
Slack 🆘  Roy Mukuye
Twitter 🐦  [Roy Mukuye](https://twitter.com/Roymkenya)
LinkedIn 🔗  [Roy Mukuye](https://www.linkedin.com/in/roy-mukuye/)

**Pull request merged**